### ISSUE TITLE
Enhancement/Suggestion/Feature Request: Exclude by Tag in Filters

### ISSUE THREAD

**Issue by pjbingham** [HearthSim/Hearthstone-Deck-Tracker#2915]
Currently we have 'And' and 'Or' available in the tag filters (which are great, by the way).  Would it be possible to add 'Not' - e.g. to view the list of all decks that _don't_ contain a particular tag (either as a separate feature, or ideally in combination with other options)?

Apologies if this has already been requested and I missed it in the search.

**Comment by azeier** [comment]
Related to #2110

**Comment by culinko** [comment]
Also similar to #1413.

**Comment by pjbingham** [comment]
I agree the use case is similar to #1413 .  Apologies I missed that one.

I did find #2110.  Agree the use case relating to operators there is also relevant, but (personally) I'm not interested in being able save combinations of tags myself - so wasn't sure whether to raise a separate case for the NOT operator (only).  Erred on the side of caution...

What's the view - is this something you're think would be good and/or looking at adding?
